src/components/ui/Image/Picture.tsx

```diff
@@ -0,0 +1,27 @@
+import type { HTMLAttributes } from 'react'
+import { memo } from 'react'
+
+import { useSources } from './useSources'
+import type { SourceOptions } from './useSources'
+import type { ImageOptions } from './useImage'
+
+interface Props extends HTMLAttributes<HTMLPictureElement> {
+  sources: SourceOptions[]
+  img: ImageOptions
+}
+
+function Picture({ sources: sourceOptions, img: imgProps }: Props) {
+  const sources = useSources(sourceOptions)
+
+  return (
+    <picture>
+      {sources.map(({ srcSet, media }, index) => {
+        return <source key={index} srcSet={srcSet} media={media} />
+      })}
+      <img {...imgProps} alt={imgProps.alt} />
+    </picture>
+  )
+}
+
+Picture.displayName = 'Picture'
```

Comment:
Do we need this for debugging purposes?

Reply:
I don't think so. Removed [here](https://github.com/vtex-sites/base.store/pull/489/commits/73f66cd686f5d1ec5379c6048cf6100405131457).